tests/test_flink_eks_tools.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+from paasta_tools.flink_eks_tools import DEFAULT_SOA_DIR
+from paasta_tools.flink_eks_tools import FlinkEksDeploymentConfig
+from paasta_tools.flink_eks_tools import load_flink_eks_instance_config
+from paasta_tools.utils import BranchDictV2
+
+
+class TestFlinkEksDeploymentConfig(unittest.TestCase):
```

Comment:
it'd be preferable to use pytest instead of unittest - we don't really use the built-in unittest library except in a vanishingly small number of ancient tests :)

Reply:
(should be pretty simple to convert this though!)

Reply:
updated